scripts/mkdoc.sh

```diff
@@ -74,38 +61,65 @@ rm -f dist/*.whl && .venv/bin/python setup.py bdist_wheel && .venv/bin/pip insta
 echo "Patching modularized Graph methods"
 .venv/bin/python3 ${SCRIPTS_FOLDER}/patch_modularized_graph_methods.py
 
-echo "Generating HTML documentation..."
-IGRAPHDIR=`.venv/bin/python3 -c 'import igraph, os; print(os.path.dirname(igraph.__file__))'`
-"$PYDOCTOR" \
-    --project-name "igraph" \
-    --project-url "https://igraph.org/python" \
-    --introspect-c-modules \
-    --make-html \
-    --html-output "${DOC_API_FOLDER}/html" \
-	${IGRAPHDIR}
-
-# PDF not supported by PyDoctor
-
-DOC2DASH=`which doc2dash 2>/dev/null || true`
-if [ "x$DOC2DASH" != x ]; then
-    echo "Generating Dash docset..."
-    "$DOC2DASH" \
-        --online-redirect-url "https://igraph.org/python/doc/api" \
-        --name "python-igraph" \
-        -d "${DOC_API_FOLDER}" \
-	    -f \
-        "${DOC_API_FOLDER}/html"
-    DASH_READY=1
-else 
-    echo "WARNING: doc2dash not installed, skipping Dash docset generation."
-    DASH_READY=0
-fi
 
-echo ""
-echo "HTML API documentation generated in ${DOC_API_FOLDER}/html"
-if [ "x${DASH_READY}" = x1 ]; then
-    echo "Dash docset generated in ${DOC_API_FOLDER}/python-igraph.docset"
+# Remove previous docs
+rm -rf "${DOC_HTML_FOLDER}"
+
+
+# Make sphinx
+echo "Generating HTML documentation..."
+if [ "x$STANDALONE" = "x1" ]; then
+  echo "Build standalone docs"
+  .venv/bin/sphinx-build \
+   -Dtemplates_path='' \
+   -Dhtml_theme='alabaster' \
+   ${DOC_SOURCE_FOLDER} ${DOC_HTML_FOLDER}
+else
+  echo "Build Jekyll-style docs"
+  .venv/bin/sphinx-build ${DOC_SOURCE_FOLDER} ${DOC_HTML_FOLDER}
+
+  if [ "x$SERVE" = "x1" ]; then
+    cd doc/html
+
+    # Copy jekyll build environment
+    cp -r ../jekyll_tools/* ./
+
+    # Install jekyll infra
+    bundle config set --local path 'vendor/bundle'
+    bundle install
+
+    # TODO: copy back?
+
+    # Build website via templates
+    bundle exec jekyll serve
+  
+  fi
 fi
 
-cd "$PWD"
+  
+
+#PWD=`pwd`
+#DOC_API_FOLDER=${ROOT_FOLDER}/doc/api
+#DOC2DASH=`which doc2dash 2>/dev/null || true`
```

Comment:
It looks like the Dash docset generation was removed without providing a replacement. Is it because `doc2dash` is not compatible with our PyDoctor-generated API docs any more after patching the stylesheets?

Reply:
this never worked on my computer, so I commented it out for now. I'll give it a more serious try but it might need a mac?

Reply:
alright, I installed `doc2dash` 2.4.0 and ran it through, I think it works but please check after merging.